### PR TITLE
My Personal Keymaps for the Planck and melody96

### PR DESCRIPTION
**Pull request by devilzmods**
## Description

Planck and melody96 keymaps for use with german keyboard layout. 
Melody96 is via compatible but expands on the default via keymap by adding combos for german umlauts and rgb layers.
Planck features a rotary encoder, custom numpad layer and layer lighting.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Me losing my keymaps every time i fuck up my dev branch

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.  
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/melody96/keymaps/dvz/config.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#define COMBO_COUNT 4
```
Comment (drashna) [inline keyboards/melody96/keymaps/dvz/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

And this needs an include guard. 
```suggestion

#pragma once

#define COMBO_COUNT 4
```

File: keyboards/melody96/keymaps/dvz/keymap.c [hunk 2]
```diff
@@ -0,0 +1,129 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/melody96/keymaps/dvz/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/planck/keymaps/dvz/config.h [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#pragma once
```
Comment (drashna) [inline keyboards/planck/keymaps/dvz/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/planck/keymaps/dvz/config.h [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    #define PING_SOUND Q__NOTE(_A6), WD_NOTE(_E7)
+#endif
+
+/*
+ * MIDI options
+ */
+
+/* Prevent use of disabled MIDI features in the keymap */
+//#define MIDI_ENABLE_STRICT 1
+
```
Comment (fauxpark) [inline keyboards/planck/keymaps/dvz/config.h:38]:
```suggestion
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**